app/client/src/components/pages/Actions/index.js

```diff
@@ -492,6 +488,42 @@ function Actions({ fullscreen, orgId, actionId, ...props }: Props) {
     setInfoHeight(node.getBoundingClientRect().height);
   }, []);
 
+  let planType = actionTypeCode;
+  if (planType === 'TMDL') {
+    planType = (
+      <>
+        Restoration Plan: <GlossaryTerm term="TMDL">TMDL</GlossaryTerm>
+      </>
+    );
+  }
+  if (planType === '4B Restoration Approach') {
+    planType = (
+      <>
+        Restoration Plan:{' '}
+        <GlossaryTerm term="4B Restoration Approach">
+          4B Restoration Approach
+        </GlossaryTerm>
+      </>
+    );
+  }
+  if (planType === 'Alternative Restoration Approach') {
+    planType = (
+      <>
+        Restoration Plan:{' '}
+        <GlossaryTerm term="Alternative Restoration Approach">
+          Alternative Restoration Approach
+        </GlossaryTerm>
+      </>
+    );
+  }
+  if (planType === 'Protection Approach') {
+    planType = (
+      <GlossaryTerm term="Protection Approach">
+        Protection Approach
+      </GlossaryTerm>
+    );
+  }
+
```

Comment:
I try not to re-assign variables if possible, so I'd probably define planType like this:

```suggestion
  const planType =
    actionTypeCode === 'TMDL' ||
    actionTypeCode === '4B Restoration Approach' ||
    actionTypeCode === 'Alternative Restoration Approach' ? (
      <>
        Restoration Plan:{' '}
        <GlossaryTerm term={actionTypeCode}>{actionTypeCode}</GlossaryTerm>
      </>
    ) : actionTypeCode === 'Protection Approach' ? (
      <GlossaryTerm term={actionTypeCode}>{actionTypeCode}</GlossaryTerm>
    ) : (
      actionTypeCode
    );
```